make_wheels.py

```diff
@@ -128,25 +187,49 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
     import subprocess; sys.exit(subprocess.call(argv))
 '''.encode('ascii')
 
+    # 1. Check for missing required licenses paths
+    missing_licenses = set(required_license_paths) - found_license_files
+    if missing_licenses:
+        print(f"\033[93mWarning: the following required license files were not found in the Zig archive: {', '.join(sorted(missing_licenses))} "
```

Comment:
I think this should be a `RuntimeError` too.

Reply:
Right, missed that! Addressed in 8d3aaa86396d2fd6b0bbc5ac71041140c14f0e4a.